### DISCUSSION TITLE
Tools for copy translations

### DISCUSSION THREAD

**Discussion by SailorMoonS** [ocombe/ng2-translate#1322] [Show and tell]
I'm working on a tools for copy translation fragments into projects. only json files.

https://github.com/SailorMoonS/so-jn2

**Comment by digitalsohailseo** [comment]
That's great i am working with many SEO tools with a handsome experience let's connect with the best [SEO consultant in Pakistan](https://www.sohailzafar.me/) for personalized insights on optimizing your digital presence.

**Comment by rbalet** [comment]
Seems note supported anymore. 
Closing this comment. 

Cheers
